lua/plugins/service.lua

```diff
@@ -8,6 +8,23 @@ local plugin = {}
 
 function plugin.onTextMessage(msg, blocks)
 
+		if blocks[1] == 'new_chat_member' then
+    local status = db:hget(('chat:%d:settings'):format(msg.chat.id), 'Clean_Msg_Service') or config.chat_settings.settings.Clean_Msg_Service
```

Comment:
> `Clean_Msg_Service`

The other options use snake_case (without capitals) so it would be good if you could use it here too. In addition, I think `clean_msg_service` sounds confusing. I would prefer `clean_service_msg`.